src/client/components/pages/revision.jsx

```diff
@@ -117,6 +117,9 @@ const EntityLink = require('../entity-link.jsx');
 				</div>
 			));
 
+			const editorTitle =
```

Comment:
Fixed in the latest commit, the title-integration pull request on data-js will have to be merged beforehand though